VENUE_IMAGES_DEV_PAGE_SPEC.md

```
@@ -0,0 +1,607 @@
+   # Monitor at /admin/oban
+   ```
+
+### Load Testing
+
+See: `test/eventasaurus_discovery/venue_images/LOAD_TESTING.md`
+
+### Cost Monitoring
+
+- Check `/admin/venue-images/stats` for real-time provider costs
+- Review metadata: `venue.image_enrichment_metadata.cost_breakdown`
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)